sdk/core/core-http/lib/util/xml.ts

```diff
@@ -51,6 +51,21 @@ const xml2jsDefaults = {
   cdata: false
 };
 
+// The default xml2js settings applicable for Atom based XML operations.
+const xml2jsSettingsForAtomXmlOperations: any = Object.assign({}, xml2jsDefaults);
+xml2jsSettingsForAtomXmlOperations.normalize = false;
+xml2jsSettingsForAtomXmlOperations.trim = false;
+xml2jsSettingsForAtomXmlOperations.attrkey = "$";
+xml2jsSettingsForAtomXmlOperations.charkey = "_";
+xml2jsSettingsForAtomXmlOperations.explicitCharkey = false;
+xml2jsSettingsForAtomXmlOperations.explicitArray = false;
```

Comment:
Except `explicitArray`, all other values above are being set to their default values.